gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -155,7 +156,7 @@ local function PreqLabels(parent, x, y)
 	tbl.info.img:SetImage("vgui/ttt/equip/icon_info")
 
 	tbl.info.Check = function(s, sel)
-		return EquipmentIsBuyable(sel, LocalPlayer():GetTeam())
+		return EquipmentIsBuyable(sel, ply)
```

Comment:
is this intedened? Shouldn't it be `ply:GetTeam()`?

Reply:
I think you saw it already, but if not have a look at the EquipmentIsBuyable function